app/src/main/java/com/byeboo/app/presentation/quest/common/writing/QuestCommonViewModel.kt

```
@@ -0,0 +1,66 @@
+@HiltViewModel
+class QuestCommonViewModel @Inject constructor(
+) : ViewModel() {
+    private val _uiState = MutableStateFlow(QuestCommonState())
+    val uiState: StateFlow<QuestCommonState> = _uiState.asStateFlow()
+
+    private val _sideEffect = MutableSharedFlow<QuestCommonSideEffect>()
+    val sideEffect: SharedFlow<QuestCommonSideEffect> = _sideEffect.asSharedFlow()
+
+    fun onBackClicked() {
+
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)